README.rst

```diff
@@ -1,6 +1,8 @@
 s2i-thoth
 ---------
 
+[![Docker Repository on Quay](https://quay.io/repository/thoth-station/s2i-thoth-ubi8-py36/status "Docker Repository on Quay")](https://quay.io/repository/thoth-station/s2i-thoth-ubi8-py36)
```

Comment:
Just one thing - this is markdown formatting, the README file is in reStructuredText.